pkg/graveler/ref/manager_test.go

```diff
@@ -177,6 +177,49 @@ func TestManager_GetBranch(t *testing.T) {
 		}
 	})
 }
+func TestManager_CreateBranch(t *testing.T) {
+	r := testRefManager(t)
+	ctx := context.Background()
+	testutil.Must(t, r.CreateRepository(ctx, "repo1", graveler.Repository{
+		StorageNamespace: "s3://",
+		CreationDate:     time.Now(),
+		DefaultBranchID:  "main",
+	}, ""))
+
+	err := r.CreateBranch(ctx, "repo1", "f1", graveler.Branch{CommitID: "c1", StagingToken: "s1"})
+	testutil.MustDo(t, "create branch f1", err)
+
+	br, err := r.GetBranch(ctx, "repo1", "f1")
+	testutil.MustDo(t, "get f1 branch", err)
+	if br == nil {
```

Comment:
note this should get an error base on code - but I guess it is always test all possibilities.